Facebook.iOS/build.cake

```diff
@@ -207,16 +215,21 @@ void CreateAndInstallPodfile (Artifact artifact)
 	if (artifact == null)
 		return;
 
-	var iOSArtifact = (iOSArtifact)artifact;
-
-	var podfilePath = $"./externals/{iOSArtifact.Id}/";
+	var podfilePath = $"./externals/{artifact.Id}/";
 	EnsureDirectoryExists (podfilePath);
 
 	var podfileBegin = new List<string> (PODFILE_BEGIN);
-	podfileBegin [0] = string.Format (podfileBegin [0], iOSArtifact.MinimunSupportedVersion);
+	podfileBegin [0] = string.Format (podfileBegin [0], artifact.MinimunSupportedVersion);
 	
 	var podfile = new List<string> (podfileBegin);
-	podfile.Add ($"\tpod '{iOSArtifact.Id}', '{iOSArtifact.Version}'");
+	podfile.Add ($"\tpod '{artifact.Id}', '{artifact.Version}'");
+
+	if (artifact.Dependencies != null && artifact.IncludeDependencies) {
+		foreach (var dep in artifact.Dependencies) {
+			podfile.Add ($"\tpod '{dep.Id}', '{dep.Version}'");
+		}
+	}
```

Comment:
Added support for including dependencies in the Podfile to avoid invalid versions.